backend/src/main/java/io/debezium/configserver/rest/ConnectorResource.java

```diff
@@ -50,23 +48,14 @@
 import io.debezium.configserver.rest.model.BadRequestResponse;
 import io.debezium.configserver.rest.model.ServerError;
 import io.debezium.configserver.service.ConnectorIntegrator;
+import static io.debezium.configserver.service.ConnectorIntegratorBase.integrators;
+import static io.debezium.configserver.service.ConnectorIntegratorBase.supportedConnectorClassnames;
```

Comment:
Hi @rk3rn3r, one thing I noticed is we are inconsistent with our placement of static imports in the UI code base, some are at the end of the import block, others at the start.  I noticed the majority are toward the end, and that is inconsistent with the code formatting used by the upstream core repository.

We should probably make sure we're following the same code format rules as the upstream repository to make it easier for IDE integrations to use the same formatting rules IMO, would you not agree?

Reply:
We are using the core repo checkstyle.xml imo. Does this not cover this? I think aligning it makes sense, although I find there are more important things than sorting imports. ;)

Reply:
> We are using the core repo checkstyle.xml imo. Does this not cover this?

That's why I asked the question because in the upstream `debezium-parent.xml` we define this:
```
                <plugin>
                    <groupId>net.revelc.code</groupId>
                    <artifactId>impsort-maven-plugin</artifactId>
                    <version>${version.impsort}</version>
                    <configuration>
                        <groups>java.,jakarta.,javax.,org.,com.,io.</groups>
                        <staticGroups>*</staticGroups>
                        <staticAfter>false</staticAfter>
                        <skip>${format.skip}</skip>
                        <removeUnused>true</removeUnused>
                    </configuration>
                </plugin>
```
So if the UI allows it to build without enforcing static imports at the top, then no the configuration isn't being applied for some reason.  If not, could you log a Jira issue so we don't loose sight of it?

Reply:
Yes, I think this is "missing".